model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/v2/tp_model.py

```diff
@@ -32,7 +32,7 @@ def get_tp_model() -> TargetPlatformModel:
     NOTE: in order to generate a target platform model with different configurations but with the same Operators Sets
     (for tests, experiments, etc.), use this method implementation as a test-case, i.e., override the
     'get_op_quantization_configs' method and use its output to call 'generate_tp_model' with your configurations.
-    This version enables metadata by default
+    This version enables metadata by default.
```

Comment:
I'm writing this comment here but it is general to this part: what about pytorch TPCs? why aren't they updated?